app/helpers/formatTime.ts

```diff
@@ -0,0 +1,36 @@
+import moment from "moment";
```

Comment:
Moment doesn't recommend using Moment anymore so maybe we should look at other libraries that are smaller (or just use native JS dates). What are we trying to display, just a string that says like "3m" or "4m". I can't imagine Grayson and I ever really putting seconds since our estimations of time aren't that exact.

Reply:
https://momentjs.com/docs/#/-project-status/